src/tools/kdb/test.cpp

```diff
@@ -437,7 +437,7 @@ bool checkArgument (std::vector<std::string> const & arguments, std::string test
 {
 	return arguments.size () == 1 || find (arguments.begin (), arguments.end (), testname) != arguments.end ();
 }
-}
+} // namespace
```

Comment:
Was this written manually by you or does the code formatter yield different results?

Reply:
Clang-Format 6.0+ adds and corrects these comments if the option [`FixNamespaceComments`](https://github.com/ElektraInitiative/libelektra/blob/master/.clang-format#L36) is set to `true`.

Reply:
see #1828